filebeat/prospector/log/config.go

```diff
@@ -62,21 +65,25 @@ type config struct {
 	recursiveGlob  bool            `config:"recursive_glob.enabled"`
 
 	// Harvester
-	BufferSize    int                     `config:"harvester_buffer_size"`
-	Encoding      string                  `config:"encoding"`
-	Backoff       time.Duration           `config:"backoff" validate:"min=0,nonzero"`
-	BackoffFactor int                     `config:"backoff_factor" validate:"min=1"`
-	MaxBackoff    time.Duration           `config:"max_backoff" validate:"min=0,nonzero"`
-	CloseInactive time.Duration           `config:"close_inactive"`
-	CloseRemoved  bool                    `config:"close_removed"`
-	CloseRenamed  bool                    `config:"close_renamed"`
-	CloseEOF      bool                    `config:"close_eof"`
-	CloseTimeout  time.Duration           `config:"close_timeout" validate:"min=0"`
-	ExcludeLines  []match.Matcher         `config:"exclude_lines"`
-	IncludeLines  []match.Matcher         `config:"include_lines"`
-	MaxBytes      int                     `config:"max_bytes" validate:"min=0,nonzero"`
-	Multiline     *reader.MultilineConfig `config:"multiline"`
-	JSON          *reader.JSONConfig      `config:"json"`
+	BufferSize int    `config:"harvester_buffer_size"`
+	Encoding   string `config:"encoding"`
+
+	ExcludeLines []match.Matcher         `config:"exclude_lines"`
+	IncludeLines []match.Matcher         `config:"include_lines"`
+	MaxBytes     int                     `config:"max_bytes" validate:"min=0,nonzero"`
+	Multiline    *reader.MultilineConfig `config:"multiline"`
+	JSON         *reader.JSONConfig      `config:"json"`
+}
+
+type LogConfig struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type LogConfig should have comment or be unexported